sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/OnSdlChoiceChosen.java

```diff
@@ -58,115 +62,125 @@ public String toString() {
 			_menuName = menuName;
 			_vrCommands = vrCommands;
 		}
-		
+
 		// Restrict no-arg constructor
 		@SuppressWarnings("unused")
-        private SdlCommand() {}
-		
+		private SdlCommand() {}
+
 		// Public Getters
 		public Integer getCommandID() {
 			return _commandID;
 		}
-		
+
 		public SdlSubMenu getParentSubMenu() {
 			return _parentSubMenu;
 		}
-		
+
 		public String getMenuName() {
 			return _menuName;
 		}
-		
+
 		public List<String> getVrCommands() {
 			return _vrCommands;
 		}
-		
+
 		public String toString() {
 			return _menuName;
 		}
 	}
-	
-	public class SdlChoice {
-		
+
+	public class SdlChoice extends RPCStruct {

```

Comment:
We shouldn't change source code just to make unit tests work. We should find a way around not changing it, or possibly removing the current tests and create tests that match this class better.